tests/manager/api/test_sirsidynix_auth_provider.py

```diff
@@ -203,6 +216,64 @@ def test_remote_patron_lookup(self, sirsi_auth_fixture: SirsiAuthFixture):
         assert patrondata.block_reason == PatronData.NO_VALUE
         assert patrondata.library_identifier == "testtype"
 
+    @pytest.mark.parametrize(
```

Comment:
@jonathangreen or @tdilauro :  I used the pytest.mark.parametrize decorator here in order to send optional parameters to the sirsi_auth_fixture test fixture.   I couldn't see a cleaner way to override the patron_status_block default that is set in the fixture.  It works, but I fear the readability is suboptimal.   I'd be curious if either of you have suggestions for accomplishing what I'm trying to do here in a more readable way.

Reply:
I made some suggestions on how you can accomplish this. I think the only change you need to the fixture is to change @dbernstein I made some suggestions how you can accomplish this. I think all the changes you need to the fixture are to change the `provider_mocked_api` function.